README.md

```diff
@@ -1 +1,149 @@
-# calculator
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="logo-removebg.png" alt="logo" width="210"  height="auto" />
+  <br/>
```

Comment:
- [ ] Nice work using the recommended README template :clap: However, the logo image is broken and it would be nice to fix it or simply remove it to make your README file look more professional